op-guide/binary-deployment.md

```diff
@@ -1,83 +1,197 @@
 ---
-title: TiDB Binary Deployment
+title: Deploy TiDB Using the Binary
 category: deployment
 ---
 
-# Binary Deployment
+# Deploy TiDB Using the Binary
 
 ## Overview
 
-A complete TiDB project contains PD, TiKV, TiDB. The start-up sequence is PD -> TiKV -> TiDB.
+A complete TiDB cluster contains PD, TiKV and TiDB. To start the database service, follow the order of PD -> TiKV -> TiDB. To close the database service, follow the reverse order of start-up.
 
-Before your start, see [TiDB Architecture](../README.md#TiDB-Architecture) and [Deployment Recommendations](../op-guide/recommendation.md).
+Before your start, see [TiDB Architecture](../README.md#TiDB-Architecture) and [Software and Hardware Requirements](recommendation.md).
 
-To quickly understand and try TiDB, see [Single Node Cluster Deployment](#single-node-deployment).
+This document describes the binary deployment of three scenarios:
 
-To try TiDB out and explore the features, see [Multiple Nodes Cluster Deployment for Test](#multiple-nodes-cluster-deployment-for-test).
+- To quickly understand and try TiDB, see [Single Node Cluster Deployment](#single-node-cluster-deployment).
+- To try TiDB out and explore the features, see [Multiple Nodes Cluster Deployment for Test](#multiple-nodes-cluster-deployment-for-test).
+- To deploy and use TiDB in production, see [Multiple Nodes Cluster Deployment](#multiple-nodes-cluster-deployment).
 
-To deploy and use TiDB in production, see [Multiple Nodes Cluster Deployment](#multiple-nodes-cluster-deployment).
+## TiDB Components and Default Ports
 
-## Download and Decompress the Official Binary Package
+### 1. TiDB Database Components (Required)
```

Comment:
Is 1. Redundant?

Reply:
Let's remove all the numbers from the titles